lib/javascript/JavascriptModulesPlugin.js

```diff
@@ -839,18 +839,25 @@ class JavascriptModulesPlugin {
 				startupSource.add(`var ${RuntimeGlobals.exports} = {};\n`);
 			}
 
-			const renamedInlinedModule = this.renameInlineModule(
-				allModules,
-				renderContext,
-				inlinedModules,
-				chunkRenderContext,
-				hooks
-			);
+			const entryIife = compilation.options.optimization.entryIife;
```

Comment:
It will be true for `"development"` and `"production"`

Maybe just have `true` and `false`?

Reply:
I should change `entryIife?: "development" | "production" | true | false;` to `	entryIife?: "true | false;`

Reply:
Done